### PR TITLE
Workflow for Deploying code

### PR DESCRIPTION
**Pull request by keshakaneria**
Issue #22 
Adding continuous integration workflow for deploying code

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    steps:
+        - uses: actions/checkout@v1
+
+        - name: Install dependencies
+          run: yarn
```
Comment (kaiiyer) [inline .github/workflows/deploy.yml]: Why using yarn ? A JS package manager for Python Project !

Comment (keshakaneria) [reply .github/workflows/deploy.yml]: So i need to remove yarn as it is not useful. What else do i need to add or remove?

Comment (kaiiyer) [reply .github/workflows/deploy.yml]: @keshakaneria Have you used travis CI ?

**Review by kaiiyer — CHANGES_REQUESTED** [review]
This GitHub Action is used to sync a directory with a remote S3 bucket. We don't want to deploy it to AWS. Can you change the code so that it uses travis CI instead ?

**Pull request closed without merging**